lib/stripe_lib/models.py

```diff
@@ -358,9 +358,11 @@ def change_subscription_plan(self, subscription_id, new_price_or_plan, prorate=T
             item = {
                 'id': stripe_subscription['items']['data'][0]['id'],
             }
-            if type(new_price_or_plan) == str:
+            if isinstance(new_price_or_plan, str):
                 item['price'] = new_price_or_plan
-            elif type(new_price_or_plan) == dict:
+            elif isinstance(new_price_or_plan, type(u'')):
+                item['price'] = new_price_or_plan
+            elif isinstance(new_price_or_plan, dict):
```

Comment:
## Reviewer Note:
Used `isinstance` instead of `type` inorder to follow same pattern for the conditional checks